### DISCUSSION TITLE
How to use Ortho4XP tiles with AutoOrtho

### DISCUSSION THREAD

**Discussion by janjacobs66** [kubilus1/autoortho#191] [Q&A]
Hi !

Apologies up front if this has been answered, but I could not find it in this discussion or elsewhere.

1.  I have 8TB of Ortho4XP tiles in 17 REZ that I want to run as 1st option, how do I do it, if possible
2. What REZ tiles/pic does AutoOrtho donwload ?

It seems that when I have my existing tiles + AutoOrtho in my custom scenery folder, that it prioritises AutoOrtho (lower REZ)
need to mention the autoortho listings are above the ortho4xp in the scenery_packs.ini

Pls advise

**Comment by Simssmith** [comment]
Are you using xOrganizer by any chance? 

Make sure in your scenery_pack.ini all AutoOrtho entries are below Ortho4XP entries.

here is example of my .ini file
```
SCENERY_PACK S:\Games\x12spare\ortho4xp\Ortho4XP13\Ortho4XP\yOrtho4XP_Overlays/
SCENERY_PACK S:\Games\x12spare\ortho4xp\Ortho4XP13\Ortho4XP\Tiles\zOrtho4XP_+32-103/
SCENERY_PACK S:\Games\x12spare\ortho4xp\Ortho4XP13\Ortho4XP\Tiles\zOrtho4XP_+33-096/
SCENERY_PACK S:\Games\x12spare\ortho4xp\Ortho4XP13\Ortho4XP\Tiles\zOrtho4XP_+33-097/
SCENERY_PACK S:\Games\x12spare\ortho4xp\Ortho4XP13\Ortho4XP\Tiles\zOrtho4XP_+33-098/
SCENERY_PACK Custom Scenery/z_asi_00/
SCENERY_PACK Custom Scenery/z_asi_05/
SCENERY_PACK Custom Scenery/z_asi_06/
SCENERY_PACK Custom Scenery/z_asi_08/
SCENERY_PACK Custom Scenery/z_autoortho/
```

**Reply by janjacobs66** [reply]
Thank you sims,

( I have xOrganiser, but never use it, manually manipulate the.ini file)

1) Idiot me !, after making the post, I experimented and rearranged the entries in the .ini file, as per your example, and testing shows its working fine

2) I'm still curious to know what the detail level of AO is ?

3) when selecting what provider to use, default / Bing etc - any recommendations ?

4) I delete the y_ortho generated by O4XP & AO, as I dont like autogen on top of my tiles

thank you for your reply

**Reply by TinkerNZ** [reply]
I believe using xOganizer is a must first of all to make enabling/disabling and ordering the scenery_packs.ini much easier.
See a dedicated thread in the XO forum on the Org.

As for 2) I believe the ZL in AO varies between ZL12 to ZL18 and is dependent on your elevation when you fly over it. For instance if you take off from an airport that area will be at highest possible ZL and as you climb to say FL30 and fly some distance the tiles way below are also cached but at a much lower ZL (since you do not need to see that detail from the high elevation) Those lower ZL tiles are faster to download and save space of course.

As for 3)  Bing seems to be the preferred provider (depending where you fly and what time of GMT) When there is high demand due to many users things may get throttled and some have suggested they got blocked even. Not my experience so far.

Finally 4) If you remove or disable y_AutoOrtho_Overlays I suspect you are inviting issues as it is part and parcel of AO functioning properly. As is the need to have the z_autoortho folder always enabled (and below the region packs in the ini order).

TinkerNZ

**Reply by kubilus1** [reply]
For question 2, The scenery is built at ZL16 generally and ZL18 at airports.  It won't try to pull in lower ZL due to height, so much as attempt to access the lowest amount of data that meets requests from XPlane at any given time.  What XPlane will do with any ortho is for things that are far, it will just read in a smaller version of an image, called a mipmap.  So this project will attempt to access map imagery that is smaller when that is called for.

For q3, yeah Bing maps is the default.  I suspect many of the user's complaining of throttling are the same ones who clear their local cache a lot.

For question 4, you don't need the y_AtoOrtho_Overlays I package.  Safe to disable or remove those.  I use SimHeaven myself for overlays.

**Reply by janjacobs66** [reply]
thank you all for the replies, that answered my questions

**Reply by TinkerNZ** [reply]
Thanks for the replies kubilus1 and for correcting my understanding on ZLs
TinkerNZ
